core/src/main/scala/org/apache/spark/sql/rapids/tool/util/stubs/ToolsSparkListenerEvent.scala

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
this feels like it shouldn't be in the stubs directory, its critical class and there is no platform specific version, correct?

Reply:
moved it to the same package.